tools/quickstart/quickstart/bridge.py

```diff
@@ -57,6 +59,21 @@ def setup_interactively(base_dir, bridge_config_file) -> None:
         Path(user_file).touch()
         return
 
+    choice = click.prompt(
+        fill(
+            "The bridge client requires access to a main chain node."
```

Comment:
I would rather ask something like Do you have a main chain node running and provide the json rpc, or run a mainchain light node.

Reply:
Also is main chain node the right term? Should we not rather say Ethereum Main Chain or Görli depending on which setup. Or Foreign chain node.

Reply:
changed.